spatialdata/_core/_rasterize.py

```diff
@@ -327,6 +335,40 @@ def _get_xarray_data_to_rasterize(
     return xdata, pyramid_scale
 
 
+def _get_corrected_affine_matrix(
+    data: Union[SpatialImage, MultiscaleSpatialImage],
+    axes: tuple[str, ...],
+    min_coordinate: ArrayLike,
+    max_coordinate: ArrayLike,
+    target_coordinate_system: str,
+) -> tuple[Affine, tuple[str, ...]]:
+    """
+    TODO: docstring
```

Comment:
Please at least add a docstring stating what the function does before merging

Reply:
added the docstring explaining why we need this function.